docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -70,7 +70,8 @@ repo-level project settings configuration. This can be learned about at
 Under project flags, admins have a number of options for users to configure what
 runs for their project. For example, admins can enable the ability to unschedule old 
 tasks if a more recent commit passes, or configure tasks to stepback on failure to 
-isolate the cause.
+isolate the cause.  Admins can also enforce [stepback bisection](./Stepback-Bisection) instead
```

Comment:
Rather than putting this here, I think another small section denoted with a `####` (just like repotracker settings below) should be added describing the stepback flags